apparmor.d/groups/lxqt/lxqt-session

```diff
@@ -0,0 +1,59 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-session
+profile lxqt-notificationd @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/gtk>
+  include <abstractions/lxqt>
+  include <abstractions/video>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/dbus-session>
+  include <abstractions/dbus-accessibility>
+  include <abstractions/gvfs-open>
+
+#  signal (receive) set=(kill, term) peer=lxqt-session,
+
+  /dev/tty						rw,
+  
+  dbus receive
+  	bus=session
+  	path="/org/freedesktop/Notifications"
+  	interface="org.freedesktop.DBus.Introspectable"
+  	peer=(name=":[0-9]*.[0-9]*"),
+  dbus send
+  	bus=session
+  	path="/org/freedesktop/Notifications"
+  	interface="org.freedesktop.Notifications"
+  	peer=(name="org.freedesktop.DBus"),
+  dbus receive
+  	bus=session
+  	path="/org/freedesktop/Notifications"
+  	interface="org.freedesktop.Notifications"
+  	peer=(name=":[0-9]*.[0-9]*"),
```

Comment:
Use the directive: `aa:dbus own bus=session name=org.freedesktop.Notifications`

Reply:
hmm, if i delete the dbus receive and send statements and replace it with
aa:dbus own bus=session name=org.freedesktop.Notifications
it states that this is unknown.
Havent worked with aa:... statements yet - or have i misunderstood what to change?

Reply:
Forgot to add the comment, it should be: `#aa:dbus own bus=session name=org.freedesktop.Notifications`